### PR TITLE
Egnyte.EgnyteDesktopApp version 3.15.5.123

### PR DESCRIPTION
**Pull request by DeusMaximus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91561)

### CODE REVIEW

File: manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Egnyte.EgnyteDesktopApp
+PackageVersion: 3.15.5.123
```
Comment (mdanish-kh) [inline manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.installer.yaml:5]:
```suggestion
PackageVersion: 3.14.5.123
```

File: manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Egnyte.EgnyteDesktopApp
+PackageVersion: 3.15.5.123
```
Comment (mdanish-kh) [inline manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.14.5.123
```

File: manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Egnyte.EgnyteDesktopApp
+PackageVersion: 3.15.5.123
```
Comment (mdanish-kh) [inline manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.yaml:5]:
```suggestion
PackageVersion: 3.14.5.123
```

File: manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet /norestart
+  SilentWithProgress: /passive /norestart
```
Comment (mdanish-kh) [inline manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.installer.yaml:16]:
```suggestion
```
These switches are already passed to the Installer as the InstallerType is set to `wix`.

https://github.com/microsoft/winget-cli/blob/c3b8d5412d90659277ab3ca6e324226b40c4679a/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L571

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
After committing the above changes, you would also have to rename the folder `3.15.5.123` -> `3.14.5.123` under `manifests/e/Egnyte/EgnyteDesktopApp/` to match PackageVersion.

You can do this here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open this PR in VSCode Online. Find your files from explorer tab on the left and rename the folder. After that you can stage and commit your changes from Source Control tab on the left (the changes are automatically pushed).

**Comment by DeusMaximus** [comment]
Well, crap. Made a typo in the wingetcreate update command with the version number.

I'll resubmit the PR, so it has the correct path as well as the version changes. I'll also do a quick edit afterwards and get rid of those installer switches. Looks like it's copying it from the previous version when creating the new manifest.

**Pull request closed without merging**